HLTrigger/Configuration/test/OnLine_HLT_GRun.py

```diff
@@ -4502,6 +4502,10 @@
 process.ZdcGeometryFromDBEP = cms.ESProducer( "ZdcGeometryFromDBEP",
   applyAlignment = cms.bool( False )
 )
+process.ZdcTopologyEP = cms.ESProducer( "ZdcTopologyEP",
```

Comment:
please do not change files in `HLTrigger/Configuration` directly, as we ultimately need to update also the configuration in confDB. Please provide the needed `ESProducer` via customization function, see  `HLTrigger/Configuration/python/customizeHLTforCMSSW.py` for examples.